src/main/java/makaroshyna/onlinebookstore/model/User.java

```diff
@@ -29,4 +41,50 @@ public class User {
     private String lastName;
 
     private String shippingAddress;
+
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
@ManyToMany by default has FetchType.LAZY, in this case, you can use EAGER, and don't need to use Query in UserRepository, but you sample is also good